content/ko/docs/concepts/services-networking/topology-aware-hints.md

```diff
@@ -13,18 +17,12 @@ weight: 45
 
 _토폴로지 인지 힌트(Topology Aware Hints)_ 는 클라이언트가 엔드포인트를 어떻게 사용해야 하는지에 대한 제안을 포함시킴으로써 
 토폴로지 인지 라우팅을 가능하게 한다. 
-이러한 접근은 엔드포인트슬라이스(EndpointSlice) 및 엔드포인트(Endpoint) 오브젝트의 소비자(consumer)가 이용할 수 있는 메타데이터를 추가하며, 
+이러한 접근은 엔드포인트슬라이스(EndpointSlice 혹은 엔드포인트(Endpoint)) 오브젝트의 소비자(consumer)가 이용할 수 있는 메타데이터를 추가하며,
```

Comment:
@jihoon-seo 위와 같은 겹괄호 발생에 대해서 의견 듣고 싶습니다.

Reply:
[원문]
> This approach adds metadata to enable consumers of EndpointSlice (or Endpoints) objects, so that traffic to those network endpoints can be routed closer to where it originated.

@Seo-yul 네, 말씀하신 대로 여기에서는 @Seo-yul 님께서 제안해 주신 대로 하면
겹괄호도 사용하지 않으면서 더 읽기 쉽고 의미 전달도 잘 될 것 같습니다. ㅎㅎ

Reply:
그러면 영문 병기 없이 한글로만 표시해도 괜찮을까요~?